packages/arb-checkpointer/checkpointing/indexedCheckpointer.go

```diff
@@ -212,7 +212,15 @@ func restoreLatestState(
 			continue
 		}
 		logger.Info().Msg("Restored state")
-		return nil
+		success := db.FlushCheckpointStorage()
+		if !success {
+			logger.Error().Msg("Failed flushing database")
+		}
+		logger.Info().Msg("Flushed database")
+		db.CloseCheckpointStorage()
```

Comment:
If we're continuing to launch after this, don't we want to keep the db open?

Reply:
forgot to push second commit